webp_backport/src/androidTest/java/de/marcreichelt/webp_backport/FileSignatureCheckerTest.java

```diff
@@ -0,0 +1,90 @@
+package de.marcreichelt.webp_backport;
+
+import android.test.AndroidTestCase;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.InputStream;
+
+import de.marcreichelt.webp_backport.test.R;
+
+public class FileSignatureCheckerTest extends AndroidTestCase {
```

Comment:
I just recognized that we can use the new awesome JVM unit tests for this test class because we do not depend on any real Android functionality (not even the native WebP library). Everything here is about byte arrays.
But we'll do that later. ;-)